hail/src/main/scala/is/hail/expr/AnnotationImpex.scala

```diff
@@ -53,6 +53,10 @@ object SparkAnnotationImpex {
         StructType(tbs.fields
           .map(f =>
             StructField(escapeColumnName(f.name), f.typ.schema, nullable = !f.typ.required)))
+    case TTuple(types, required) =>
```

Comment:
there's a slightly easier fix -- change the `TStruct` above to `TBaseStruct`, which is the common superclass of TStruct and TTuple. The variable name `tbs` also indicates this is what we meant!

Reply:
Yeah, totally agree with that. Thanks.